framework_tvb/tvb/config/logger/cluster_handler.py

```diff
@@ -42,44 +42,42 @@
 class ClusterTimedRotatingFileHandler(MemoryHandler):
     """
     This is a custom rotating file handler which computes the name of the file depending on the 
-    execution environment (web node or cluster node)
+    execution environment (web node, cluster node or hpc)
     """
 
     # Name of the log file where code from Web application will be stored
     WEB_LOG_FILE = "web_application.log"
 
     # Name of the file where to write logs from the code executed on cluster nodes
-    CLUSTER_NODES_LOG_FILE = "operations_executions.log"
+    ASYNC_OP_LOG_FILE = "operations_executions.log"
 
     # Size of the buffer which store log entries in memory
     # in number of lines
     BUFFER_CAPACITY = 20
 
-
     def __init__(self, when='h', interval=1, backupCount=0):
         """
         Constructor for logging formatter.
         """
         # Formatting string
         format_str = '%(asctime)s - %(levelname)s'
         if TvbProfile.current.cluster.IN_OPERATION_EXECUTION_PROCESS:
-            log_file = self.CLUSTER_NODES_LOG_FILE
+            log_file = self.ASYNC_OP_LOG_FILE
             if TvbProfile.current.cluster.IS_RUNNING_ON_CLUSTER_NODE:
                 node_name = TvbProfile.current.cluster.CLUSTER_NODE_NAME
                 if node_name is not None:
                     format_str += ' [node:' + str(node_name) + '] '
             else:
                 format_str += ' [proc:' + str(os.getpid()) + '] '
```

Comment:
I don't think that anyone is ever looking at the node_name or PID, but maybe we could loge the operation ID instead both for the cluster and the HPC backends ?

Reply:
Would probably make more sense but I don't have a quick solution

Reply:
No problem, we can leave it for another refactoring round